process/README.md

```diff
@@ -129,9 +129,9 @@ The configuration for this automation is in
 _More on the Prow process in general
 [is available in the k8s docs](https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#the-code-review-process)._
 
-The Tekton community promotes company diversity as a best practice for pull request.
-This means that, where possible, one of the reviewers of a pull request and the author
-should be affiliated to different organizations.
+The Tekton community promotes company diversity as a best practice for pull requests.
+We encourage for large or significant changes, by maintainer discretion, that pull
```

Comment:
Thanks for this @jerop - sorry I missed the conversation in the meeting.

I'm not completely convinced by the "large or significant changes" - that seems to be defining a set of criteria to be applied to decide when company diversity is required, which seems inconsistent with the "maintainer" discretion.

The "where possible" part in the original statement is probably less than ideal - but I personally would prefer to keep the company diversity best practices intact, and not weakened by some kind of rules.
 
Note that it is defined today as a best practice and not as a strict rule, so maintainer discretion is already implied, but I don't mind making that more explicit. In some projects, for instance, company diversity might be difficult to achieve today, which is fine, but in future more companies might start contributing, and so the best practices would become applicable to them too.

In pipeline and TEPs we have maintainers from many companies, so company diversity should be the norm. If a maintainer considers that a PR can be merged without company diversity we trust their judgement. I would welcome a small comment on the PR in such cases.

Reply:
Thank you for the input @afrittoli!

That's right, large or significant is subject to interpretation -- removed this qualification 

Updated this to just make the maintainer discretion explicit